src/main/java/com/coveros/selenified/Browser.java

```diff
@@ -30,61 +37,85 @@
  *
  * @author Max Saperstone
  * @version 3.0.4
- * @lastupdate 5/16/2018
+ * @lastupdate 1/12/2019
  */
 public class Browser {
+
+    private static final Logger log = Logger.getLogger(Browser.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "log" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1068)